firmware/components/uart_imu/uart_imu.c

```diff
@@ -247,32 +247,34 @@ int imu_init()
   const char cmd5[8] = {0x75, 0x65, 0x01, 0x02, 0x02, 0x06, 0xE5, 0xCB};
   const char cmd6[13] = {0x75, 0x65, 0x0C, 0x07, 0x07, 0x40, 0x01, 0x00, 0x0E, 0x10, 0x00, 0x53, 0x9D}; // 921600 bauds
 
-  printf("sending new baud rate setting to the IMU \n");
-  vTaskDelay(30);
   uart_write_bytes(UART_NUM, cmd0, sizeof(cmd0));
-  vTaskDelay(10);
+
+  printf("sending new baud rate setting to the IMU \n");
```

Comment:
to be removed

Reply:
This message was in the original code as well (see the diff). I can remove it but as it was there before, I thought to keep it.